ui/webui/test/helpers/installer.py

```diff
@@ -26,19 +26,41 @@ class Installer():
     def __init__(self, browser):
         self.browser = browser
 
-    def begin_installation(self):
+    def begin_installation(self, should_fail=False, confirm_erase=True):
+        current_step_id = self.get_current_page_id()
         self.browser.click("button:contains('Begin installation')")
-        self.browser.click("#installation-review-disk-erase-confirm")
 
-    def next(self):
+        self.browser.wait_visible("#installation-review-disk-erase-confirm")
```

Comment:
You don't need wait_visible here. 

b.click has the wait_visible check inside.

Reply:
https://github.com/cockpit-project/cockpit/blob/main/test/common/testlib.py#L289